.circleci/config.yml

```diff
@@ -45,6 +50,21 @@ jobs:
           paths:
             - build/*
             - s3_website.yml
+  build_pdf:
+    working_directory: ~/work
+    docker:
+      - image: tianon/latex
```

Comment:
Looks like tianon's image does not have some of the tools (e.g., git, ssh) that CircleCI needs. See https://circleci.com/docs/2.0/custom-images/#adding-required-and-custom-tools-or-files for that list. I tried installing them after the build but that didn't work.

Let's try `schickling/latex`. It's seems fairly popular and it built in my branch.

Reply:
Thank you for trying this out and fixing it! I will make the changes here :smile:

Reply:
Thank you for trying this out and fixing it! I will make the changes here :smile: